docs/exercises/01-gant-setting-up-the-environment.md

```diff
@@ -0,0 +1,124 @@
+# &Uuml;bung 1 Ganttproject:  Setup der Umgebung
+
+## Administratives
+
+*Abgabe:* Bearbeiten bis 27. September 2017. Abgabe zusammen mit n&auml;chster &Uuml;bung bis 3. Oktober 2017 via Pullrequest
+ auf Github. Details finden Sie auf dem n&auml;chsten &Uuml;bungsblatt.
+
+## Einf&uuml;hrung
+
+F&uuml;r die meisten &Uuml;bungen in diesem Kurs werden wir das Open Source Projekt [Ganttproject](http://www.ganttproject.biz/) verwenden. 
+In dieser ersten &Uuml;bung werden Sie die Umgebung (Werkzeuge, Entwicklungsumgebung) aufsetzten, damit sie im folgenden &Auml;nderungen und Erweiterungen im Code vornehmen k&ouml;nnen. 
+
+## Vorbereitung
+
+Falls sie noch keinen Benutzeraccount auf [Github](https://www.github.com) haben, erstellen sie bitte einen. Teilen sie mir Ihren Benutzernamen via Mail mait(marcel.luethi@unibas.ch), damit wir Ihnen Zugriff auf das Code-Repository geben k&ouml;nnen.  
+
```

Comment:
Vielleicht hier sagen, dass man sich optional mit dem [Github Education Pack](https://education.github.com/pack) auch einen privaten fork einrichten kann

Reply:
@silvanheller  Ich finde die Idee gut, auf das Github Educational Pack hinzuweisen. Hier ist es aber nicht notwendig, da der Fork ja sowieso privat bleibt. Vielleicht k&ouml;nnten wir in deiner Pr&auml;sentation darauf hinweisen?

Reply:
@marcelluethi einverstanden